beta-20170406.md

```diff
@@ -0,0 +1,76 @@
+---
+title: What&#39;s New in beta-20170406
+toc: false
+summary: Additions and changes in CockroachDB version beta-20170406.
+---
+
+## Apr 6, 2017
+
+Get future release notes emailed to you:
+<div class="hubspot-install-form install-form-1 clearfix">
+    <script>
+        hbspt.forms.create({
+            css: '',
+            cssClass: 'install-form',
+            portalId: '1753393',
+            formId: '39686297-81d2-45e7-a73f-55a596a8d5ff',
+            formInstanceId: 1,
+            target: '.install-form-1'
+        });
+    </script>
+</div>
+
+### Binaries
+
+<div id="os-tabs" class="clearfix">
+    <a href="https://binaries.cockroachdb.com/cockroach-beta-20170406.darwin-10.9-amd64.tgz"><button id="mac" data-eventcategory="mac-binary-release-notes">Mac</button></a>
+    <a href="https://binaries.cockroachdb.com/cockroach-beta-20170406.linux-amd64.tgz"><button id="linux" data-eventcategory="linux-binary-release-notes">Linux</button></a>
+</div>
+
+### Backwards-Incompatible Changes
+
+- The implementation of indexing for [collated string columns](collate.html) has changed. Any secondary indexes containing collated strings must be [dropped](drop-index.html) before upgrading to this version and then [re-created](create-index.html) after upgrading. Note that this applies only to secondary indexes, not to primary indexes. [#14528](https://github.com/cockroachdb/cockroach/pull/14528)
+- Any row with a `-0` [`FLOAT`](float.html) value in a primary key or index must be [deleted](delete.html) before upgrading to this version and then [re-inserted](insert.html) after upgrading. Otherwise, the row containing `-0` will show up during table scans, but it will not be possible to delete it or select it as part of an index scan. [#14528](https://github.com/cockroachdb/cockroach/pull/14528)
```

Comment:
I'd add this text at the end:

Due to a bug in previous versions, it is not possible to use a `SELECT` to find these values of `-0` if they are in an index. If you suspect these values are present, an alternative is to use `cockroach dump` to backup the table, drop it, recreate it and its indexes, then re-insert the data from the dump.